sites/search/search-en.html

```diff
@@ -0,0 +1,33 @@
+---
+{
+	"altLangPage": "search-fr.html",
+	"dateModified": "2023-07-17",
+	"description": "Documentation on how to use search bar.",
+	"language": "en",
+	"title": "GC Site Search Bar example"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<p>To see a practical implementation of the GC site search bar in action, look no further than the site header. </p>
+
+<h2>Code</h2>
+<pre><code>
+	&lt;section class="col-xs-12 col-sm-5 col-md-4"&gt;
+		&lt;form action="#" method="post" name="cse-search-box" role="search" class="form-inline"&gt;
+			&lt;div class="form-group wb-srch-qry"&gt;
+				&lt;label for="wb-srch-q" class="wb-inv"&gt;Search Canada.ca&lt;/label&gt;
+				&lt;div class="input-group"&gt;
```

Comment:
Suggestion, instead of hard coding the source code, the best will be to create code sample file (EN and FR) in the "sites/search/samples" folder and then include them.

Jekyll include example, like this example https://github.com/wet-boew/GCWeb/blob/master/sites/feedback/page-feedback-en.html#L30

```
{% highlight text %}
{% include_relative sample/site-search-en.html %}
{% endhighlight %}
```